app/components/sdg/related_list_selector_component.rb

```diff
@@ -0,0 +1,31 @@
+class SDG::RelatedListSelectorComponent < ApplicationComponent
+  attr_reader :f
+
+  def initialize(form)
+    @f = form
+  end
+
+  def checked?(code)
+    f.object.sdg_goals.find_by(code: code).present?
+  end
+
+  def sdg_related_suggestions
+    goals.map do |goal|
+      [goal, *goal.targets.sort]
+    end.flatten.map do |goal_or_target|
+      display_text = goal_or_target.class.name == "SDG::Goal" ? "SDG#{goal_or_target.code}" : "#{goal_or_target.code}"
```

Comment:
Forgot one thing :relieved:.

Should we use internationalization here? So, instead of "SDG1", we get "ODS1" in Spanish (for instance).